core/src/main/java/com/amplifyframework/datastore/DataStoreChannelEventName.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package com.amplifyframework.datastore;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+
+import com.amplifyframework.hub.HubCategory;
+import com.amplifyframework.hub.HubChannel;
+import com.amplifyframework.hub.HubEvent;
+
+import java.util.Objects;
+
+/**
+ * An enumeration of the names of events relating the the {@link DataStoreCategory},
+ * that are published via {@link HubCategory#publish(HubChannel, HubEvent)} on the
+ * {@link HubChannel#DATASTORE} channel.
+ */
+public enum DataStoreChannelEventName {
```

Comment:
I like this but do we have a similar structure on iOS that we could align with names? @palpatim @drochetti

Reply:
Good point - since this is API surface, we should get the names and behavior aligned across platforms.